chain/network/src/routing/routing_table_view.rs

```diff
@@ -12,24 +11,28 @@ use std::sync::Arc;
 use tracing::warn;
 
 const ANNOUNCE_ACCOUNT_CACHE_SIZE: usize = 10_000;
-const ROUND_ROBIN_MAX_NONCE_DIFFERENCE_ALLOWED: usize = 10;
-const ROUND_ROBIN_NONCE_CACHE_SIZE: usize = 10_000;
+const LAST_ROUTED_CACHE_SIZE: usize = 10_000;
+
+type RoutingTable = Arc<HashMap<PeerId, Vec<PeerId>>>;
```

Comment:
We have another `pub type RoutingTable = HashMap<PeerId, Vec<PeerId>>;` in `graph_with_cache`. Could use that here given the `pub`? It also seems fit to let the `Arc` outside

Reply:
done